app/src/test/java/com/capstone/weatherapp/DetailsViewModelTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.capstone.weatherapp
+
+import io.mockk.coEvery
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class DetailsViewModelTest {
```

Comment:
also, let's move the `convertTime()` function from your DetailsFragment and put it in the DetailsViewModel. This is logic that should be unit tested, but currently can't be since we don't unit test the fragment. Once we move that method into the ViewModel however, we are able to test it